test.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+function anotherStep(string $input) {
+    return $input;
+}
+
+$input = isset($_POST['bar']) ? $_POST['bar'] : '';
+
+$unmodifiedInput = anotherStep($input);
+
+print($unmodifiedInput);
+
```

Comment:
TODO: Remove. This is just the sample file that I used for taint-flow vulnerability scanning.